wgscovplot/tools/mosdepth/flu.py

```diff
@@ -1,204 +1,71 @@
 from pathlib import Path
+from collections import defaultdict
 import logging
 from typing import List, Tuple, Dict
 import base64
-
 import pandas as pd
-from Bio import SeqIO
+from pydantic.main import BaseModel
 
-from wgscovplot.tools.mosdepth import SAMPLE_NAME_CLEANUP
-from wgscovplot.util import find_file_for_each_sample
-from wgscovplot.tools.mosdepth import logger, read_mosdepth_bed, depth_array, count_positions, \
+from wgscovplot.tools.mosdepth import read_mosdepth_bed, depth_array, count_positions, \
     get_interval_coords_bed, get_genome_coverage, get_genome_length, MosdepthDepthInfo
 
+
 logger = logging.getLogger(__name__)
 
 TOP_REFERENCE_PATTERNS = [
-    '**/reference_sequences/**/*.topsegments.csv',
+    '*.topsegments.csv',
```

Comment:
Updated for https://github.com/peterk87/wgscovplot/pull/3#discussion_r1420841328